action/README.md

```diff
@@ -19,7 +19,7 @@ Actions" deployment mode. You can do this in your project settings on
 https://dash.deno.com.
 
 ```yml
-job:
+jobs:
```

Comment:
I guess we need another key for job name (like `deploy` or anything) if we change this to `jobs`

Reply:
The structure needs to be:
```yaml
jobs:
  job-name:
    permissions: ...
    steps: ...